README.md

```diff
@@ -2,548 +2,282 @@
 
 # debug.rb
 
-This library provides debugging functionality to Ruby (MRI) 2.6 and later.
-
-This debug.rb is replacement of traditional lib/debug.rb standard library which is implemented by `set_trace_func`.
```

Comment:
I removed this part because not many developers don't even know about the old `debug.rb`